### ISSUE TITLE
Documentation doesn't mention save all the tabs VS extension

### ISSUE THREAD

**Issue by jon49** [TeaDrivenDev/Amagatsha#4]
Pretty awesome you wrote this. In the documentation it sounds like you haven't heard of save all the tabs extension. A great little extension where you can do what you outlined here.

https://marketplace.visualstudio.com/items?itemName=eamodio.SaveAlltheTabs

**Comment by TeaDrivenDev** [comment]
You are correct, I was not aware of that extension; that seems to be pretty much what I originally wanted, with the additional shortcut of using the current branch for the name (which should be easy to add). Judging from the issues, people seem to have some problems with it, but it seems worth checking out. Thanks for mentioning it!
